content/docs/user-guide/project-structure/dvcyaml-files.md

```diff
@@ -314,6 +314,13 @@ will be listed by `dvc params diff`).
 
 ### Dictionary unpacking
 
+<admon type="tip">
+
+This is most useful for non-Python commands since our Python API includes
+`dvc.api.params_show()` to load parameters from code.
```

Comment:
This is a good tip and I like having another place to suggest `dvc.api.params_show()`, but this came from an initial request that was Python-based: https://github.com/iterative/dvc/issues/6107. It might be good to specify that it's also useful if you already have a script that accepts many arguments.

Reply:
Updating...